source/simulator/introspection.cc

```diff
@@ -333,6 +333,46 @@ namespace aspect
 
 
 
+  template <int dim>
+  std::vector<unsigned int>
+  Introspection<dim>::get_compositional_field_base_element_indices() const
```

Comment:
maybe this would help a bit to avoid misunderstanding this function:
```suggestion
  Introspection<dim>::get_composition_base_element_indices() const
```